docs/attributes-registry/messaging.md

```diff
@@ -36,7 +36,7 @@
 | `messaging.rocketmq.message.tag` | string | The secondary classifier of message besides topic. | `tagA` |
 | `messaging.rocketmq.message.type` | string | Type of message. | `normal` |
 | `messaging.rocketmq.namespace` | string | Namespace of RocketMQ resources, resources in different namespaces are individual. | `myNamespace` |
-| `messaging.system` | string | A string identifying the messaging system. | `kafka`; `rabbitmq`; `rocketmq`; `activemq`; `AmazonSQS` |
```

Comment:
nit: why is `AmazonSQS` being removed as an example as part of this PR?

Reply:
hehe, I just asked the same thing 😁https://github.com/open-telemetry/semantic-conventions/pull/490#discussion_r1389390975